### PR TITLE
Devx 24n

### PR DESCRIPTION
**Pull request by resmaa**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1000#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+5. **Review** the configuration and click on the **Finish Configuration** button.
+
+
+The Palette Dev Engine App Profile is now created and and can be used for Apps deployment. To edit or delete the App Profile, navigate to the **App Profile** page. Select the desired App Profile and click on the  **Settings** button at the top righthand side. 
+
+<br />
+
+## Validation
+
+You can validate the App Profile is avaiable and ready for us by visiting the App Profiles page. In the App Profiles page, you will find your App Profile listed.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/00-create-app-profile.md]:
```suggestion
You can validate the App Profile is available and ready for use by visiting the App Profiles page. On the App Profiles page, you will find your App Profile listed.
```

File: content/docs/04.5-devx/05-app-profile/05-versioning-app-profile.md [hunk 2]
```diff
@@ -48,18 +48,48 @@ The following attributes are non-editable during versioning:
...
-* From the **Drop Down** menu next to the app profile name select the **Create New Version**.
+* From the **Drop Down** menu next to the App Profile name, select the **Create New Version**.
 
 
-* Give the version number as per the format described.    
+* Give the version number per the semantic format described above.
 
 
-* **Confirm** the version number to complete the versioning wizard.
+* Click **Confirm** to the wizard. The UI will give a versioning successful message.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/05-versioning-app-profile.md:63]:
```suggestion
* Click on **Confirm** to complete the wizard. The UI will return a versioning successful message.
```

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 661ab55aa6ce58644aeef97aa73392d957e3a86a | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1000/librarium_build/1595835090208821248) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04.5-devx/05-app-profile/05-versioning-app-profile.md [hunk 3]
```diff
@@ -48,18 +48,48 @@ The following attributes are non-editable during versioning:
...
 
-The new version of the app profile will be saved under the same app profile name. Users can select the app profile by selecting the name and version number for deployments.
 
+* The App Profile list will display the new version of the App Profile. The users can select the app profile by selecting the name and version number for deployments.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/05-versioning-app-profile.md:66]:
```suggestion
* The App Profile list will display the new version of the App Profile. Users can select the App Profile by selecting the name and version number for deployments.
```

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1000#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/rename/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md [hunk 4]
```diff
@@ -42,10 +42,11 @@ To create an App Profile, follow the steps below.
 
 4. **Add Service**: Select a service to start configuring the App Profile from the list of services available. The service you select will require you provide information to generate the configuration. The following is an explanation of the types of services available. 
 
-  * Container Deployment: [Containers](https://www.docker.com/resources/what-container/) are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment. Review the [Palette Container Deployment](/devx/app-profile/container-deployment) to learn more about container deployment.
+  * [Container Deployment](/devx/app-profile/container-deployment): Containers are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:45]:
```suggestion
  * Container Deployment: [Containers](https://www.docker.com/resources/what-container/) are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment. Review the [Palette Container Deployment](/devx/app-profile/container-deployment) to learn more about container deployment.
```
Re-adding

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md [hunk 5]
```diff
@@ -42,10 +42,11 @@ To create an App Profile, follow the steps below.
...
  
 
- * [Helm](/devx/registries/helm-registry#palettehelmregistry): Helm chart repositories are availabe for use. You can add additional Helm registries to Palette, visit the [Palette Helm Registry](/devx/registries/helm-registry#palettehelmregistry) documentation page for guidance..
+ * [Helm](/devx/registries#palettehelmregistry): Palette App profile Pack from Helm registry charts. The public Palette Pack registry and a few popular Helm chart repositories are already available out of the box. Add additional pack registries or public or private chart registries to Palette.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:48]:
```suggestion
 * [Helm](/devx/registries/helm-registry#palettehelmregistry): Helm chart repositories are available for use. You can add additional Helm registries to Palette, visit the [Palette Helm Registry](/devx/registries/helm-registry#palettehelmregistry) documentation page for guidance.
```
Re-adding previous commit from feedback

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1000#pullrequestreview-1193493221" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1000#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/rename/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | f486abc7f480b2a3d973118bc4058adc0901eecb | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1000/librarium_build/1595843163992363008) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**